test/functional/domains_controller_test.rb

```diff
@@ -1,15 +1,11 @@
 require 'test_helper'
+require 'functional/shared/basic_rest_response_test'
```

Comment:
Just repeating an earlier unanswered comment:

> It might be easier to include this straight into the TestCase class in test_helper, or at least require the file from test_helper as it's used so much.